packages/block-editor/src/components/letter-spacing-control/index.js

```diff
@@ -15,12 +15,18 @@ import useSetting from '../../components/use-setting';
 /**
  * Control for letter-spacing.
  *
- * @param {Object}   props          Component props.
- * @param {string}   props.value    Currently selected letter-spacing.
- * @param {Function} props.onChange Handles change in letter-spacing selection.
- * @return {WPElement}                      Letter-spacing control.
+ * @param {Object}   props                      Component props.
+ * @param {string}   props.value                Currently selected letter-spacing.
+ * @param {Function} props.onChange             Handles change in letter-spacing selection.
+ * @param {boolean}  props.__unstableInputWidth Input width to pass through to inner UnitControl.
```

Comment:
By looking at https://github.com/WordPress/gutenberg/blob/0160b7f1ccdc806a64eb111b36b4fd9c328ad49a/packages/components/src/input-control/types.ts, it seems like this prop should be of type `string | number | undefined` (and not `boolean`) — is this on purpose?

Reply:
I've created follow-up PR to address this: https://github.com/WordPress/gutenberg/pull/36367.

To correct the `boolean` type here but still override the default I needed to use `null`. For backwards compatibility, we need to provide a default width so setting the `__unstableWidth` prop to `undefined` doesn't work when wishing to restore full width to the control under the Typography `ToolsPanel`.